release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -3452,7 +3455,7 @@ To update an existing {product-title} 4.11 cluster to this latest release, see x
 
 Issued: 2023-04-13
 
-{product-title} release 4.11.36 is now available. The bug fix that is included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:1733[RHBA-2023:1733] advisory. There are no RPM packages for this update.
+{product-title} release 4.11.36 is now available. The bug fix that is included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:1733git[RHBA-2023:1733] advisory. There are no RPM packages for this update.
```

Comment:
Ooopise! 
s/[release 4.11.36 is now available.](https://access.redhat.com/errata/RHBA-2023:1733git/https://access.redhat.com/errata/RHBA-2023:1733

Reply:
Case of the Mondays.